src/System.Management.Automation/engine/remoting/common/RemoteSessionNamedPipe.cs

```diff
@@ -1123,7 +1132,7 @@ protected virtual NamedPipeClientStream DoConnect(int timeout)
     /// creates the named pipe name and initiates connection to
     /// target named pipe server.
     /// </summary>
-    internal sealed class RemoteSessionNamedPipeClient : NamedPipeClientBase
+    public sealed class RemoteSessionNamedPipeClient : NamedPipeClientBase
```

Comment:
This is only public for the test?

Reply:
Correct.

Reply:
My preference would be to not have this public as an API we would support if it's only for test purposes.  Can we use reflection instead for testing?

Reply:
I'll just copy the code.  Making the type public may be useful, but I don't really have a scenario in mind.